docs/content/dagster-cloud/developing-testing/environment-variables-and-secrets.mdx

```diff
@@ -133,63 +120,306 @@ In the following table, we compare the supported methods for managing and passin
+     config:
+       cluster: ${ConfigCluster}
+       subnets: [${ConfigSubnet}]
+       service_discovery_namespace_id: ${ServiceDiscoveryNamespace}
+       execution_role_arn: ${TaskExecutionRole.Arn}
+       task_role_arn: ${AgentRole}
+       log_group: ${AgentLogGroup}
+       env_vars:
+         - SNOWFLAKE_USERNAME=dev
+         - SNOWFLAKE_PASSWORD       ## pulled from agent environment
+   ' > $DAGSTER_HOME/dagster.yaml && cat $DAGSTER_HOME/dagster.yaml && dagster-cloud agent run"
+   ```
 
-  </TabItem>
-  <TabItem name="dagster-cloud CLI (Serverless)">
-<Note>Only supported for Serverless deployments.</Note>
+8. When finished, click the **Create Stack** button:
 
-If you're using a git provider like Bitbucket, you can use the [dagster-cloud CLI](/dagster-cloud/developing-testing/dagster-cloud-cli) to pass in environment variables with the `--env` flag.
+   <Image
+   alt="Highlighted Create Stack button in the AWS Console"
+   src="/images/dagster-cloud/developing-testing/environment-variables/aws-ecs-save-template.png"
+   width={1566}
+   height={375}
+   />
 
-This flag accepts `KEY=VALUE` pairs. For example:
+9. You'll be redirected back to the **Update stack** wizard, where the new template will be populated. Click **Next**.
 
-```shell
-dagster-cloud serverless deploy --env DATABASE_USERNAME=hooli_testing
+10. Continue to click **Next** until you reach the **Review** page.
+
+11. Click **Submit** to update the stack.
+
+</TabItem>
+<TabItem name="Docker">
+
+To make environment variables accessible to a full deployment with a Docker agent, you'll need to modify your project's `dagster.yaml` file.
+
+In the `user_code_launcher` section, add an `env_vars` property as follows:
+
+```yaml
+user_code_launcher:
+module: dagster_cloud.workspace.docker
+class: DockerUserCodeLauncher
+config:
+  networks:
+    - dagster_cloud_agent
+  env_vars:
+    - SNOWFLAKE_PASSWORD # value pulled from agent's environment
+    - SNOWFLAKE_USERNAME=dev
 ```
 
-  </TabItem>
-  <TabItem name="Agent configuration (Hybrid)">
-<Note>Only supported for Hybrid deployments.</Note>
+In `env_vars`, specify the environment variables as keys (`SNOWFLAKE_PASSWORD`) or key-value pairs (`SNOWFLAKE_USERNAME=dev`). If only `KEY` is provided, the value will be pulled from the local (agent's) environment.
+
+</TabItem>
+<TabItem name="Kubernetes">
+
+To make environment variables to a full deployment with a Kubernetes agent, you'll need to modify and upgrade the Helm chart's `values.yaml`.
 
-For Dagster Cloud Hybrid deployments, making environment variables accessible is accomplished by adding them to your agent's configuration in your project's <code>dagster_cloud.yaml</code> file. How the <code>container_context</code> property is configured depends on the agent type.
+1. In `values.yaml`, add or locate the `workspace` value.
 
-Refer to the configuration reference for your agent type for more info and examples:
+2. Add an `envVars` property as follows:
```

Comment:
similar to ECS - I'd probably first recommend that somebody put a k8s secret here rather than env var key-value pairs. the nice thing about k8s secrets is that they stay more secret, and you can add and remove env vars without having to go back and do this again

Reply:
I'm gonna punt on this for a follow up PR, but it can definitely be added in